doc/reference/aliases/default.rst

```diff
@@ -0,0 +1,26 @@
+@default
+========
+
+This alias corresponds to the default argument for ``dune build``: ``dune
+build`` is equivalent to ``dune build @@default``. Similarly, ``dune build
```

Comment:
I think it's surprising that the documentation about `@default` suddenly talks about `@@default`.